data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -116,9 +116,9 @@
     "armor": [ { "encumbrance": 15, "coverage": 70, "covers": [ "arm_l", "arm_r", "hand_l", "hand_r" ] } ]
   },
   {
-    "id": "ice_gliders",
+    "id": "ice_gliders_on",
```

Comment:
Don't change ids it will break saves where the itme is present

Reply:
sure, didn't think about it